prosemirror-jdita/tsconfig.json

```diff
@@ -65,6 +65,7 @@
     // "emitDecoratorMetadata": true,         /* Enables experimental support for emitting type metadata for decorators. */
 
     /* Advanced Options */
+    "skipLibCheck": true,                     /* Skip type checking of declaration files. */
```

Comment:
Typescript compiler was showing error from node_modules.
this will prevent it from scanning those files